src/audioin.js

```diff
@@ -31,7 +31,7 @@ p5sound.inputSources = [];
  *
  *   function setup(){
  *    let cnv = createCanvas(100, 100);
- *    cnv.mousePressed(userStartAudio);
+ *    cnv.mousePressed(useraudiocontextStartAudio);
```

Comment:
not sure why this changed (or if you meant to change it)?

Reply:
oh! i dont need to make that happen! 
no worries , i ll be correcting it !